project/src/utils/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { Offer } from '../types/offer';
+import { SortTypes } from './const';
+
+const MAX_RATING = 5;
+
+export const getRatingPercent = (rating: number) => (rating / MAX_RATING) * 100;
+
+export const getFormatDate = (date: string) => {
+  const currentDate = new Date(date);
+  const getMonth = currentDate.toLocaleString('en', { month: 'long' });
+  const getYear = currentDate.getFullYear();
+
+  return `${getMonth} ${getYear}`;
+};
+
+const sortByPriceLow = (offers: Offer[]) => offers.sort((offerOne, offerTwo) => offerOne.price - offerTwo.price);
```

Comment:
Как то более принято сокращать a,b
```suggestion
const sortByPriceLow = (offers: Offer[]) => offers.sort((a, b) => a.price - b.price);
```